File: sources/DeviceMethods_CS/EditTextNoDrag.cs
namespace com.lab714.dreamscreenv2.devices
{
	using Context = android.content.Context;
	using AppCompatEditText = androidx.appcompat.widget.AppCompatEditText;
	using AttributeSet = android.util.AttributeSet;
	using DragEvent = android.view.DragEvent;

	public class EditTextNoDrag : AppCompatEditText
	{
		public EditTextNoDrag(Context context) : base(context)
		{
		}

		public EditTextNoDrag(Context context, AttributeSet attrs) : base(context, attrs)
		{
		}

		public EditTextNoDrag(Context context, AttributeSet attrs, int defStyleAttr) : base(context, attrs, defStyleAttr)
		{
		}

		public virtual bool onDragEvent(DragEvent @event)
		{
			return false;
		}
	}

}

File: sources/APK_Decompiled_to_java/DSAPK/src/main/java/com/lab714/dreamscreenv2/devices/EditTextNoDrag.java
package com.lab714.dreamscreenv2.devices;

import android.content.Context;
import androidx.appcompat.widget.AppCompatEditText;
import android.util.AttributeSet;
import android.view.DragEvent;

public class EditTextNoDrag extends AppCompatEditText {
    public EditTextNoDrag(Context context) {
        super(context);
    }

    public EditTextNoDrag(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    public EditTextNoDrag(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }

    public boolean onDragEvent(DragEvent event) {
        return false;
    }
}
